8.1.1V8/8.1.1V8/Account.cs

```
@@ -0,0 +1,68 @@
+﻿using System;
+
+namespace _8._1._1V8
+{
+    public class Account
+    {
+        string name;
+        int id;
+        double accrualPercentage;
+        public double bynSum { get; private set; }
+        double conversionRate;
+
+        public Account(string name, double accrualPercentage, double conversionRate)
+        {
+            Random random = new Random();
+            this.name = name;
+            id = random.Next(Convert.ToInt32(DateTime.Now.Second));
+            this.accrualPercentage = accrualPercentage;
+            this.conversionRate = conversionRate;
+        }
+
+        public void ChangeOwner(string newOwner)
+        {
+            name = newOwner;
+        }
+
+        public void Deposit(double sum)
+        {
+            if (sum < 0)
+            {
+                Withdraw(Math.Abs(sum));
```

that's not a datetime, these operation is more specific... Depositing resulting in a withdraw can cost hell a lot of money. Better to throw an error.